assets/dash.css

```diff
@@ -2338,3 +2338,31 @@ margin-bottom: 35px;
 footer .footer-bottom{
   background: #1f1f1f!important;
 }
+
+/* DashDirect Homepage Banner */
+.home-banner-btn{
+  border-color: #008de4;
+  margin-top: 45px;
+  font-size: 1.1em;
+  box-shadow: 5px 5px 32px 5px rgba(0,0,0,0.37);
+}
+
+.dash-direct-homehero{
+  max-width: 600px;
+  margin: 0 auto;
+  font-size: 1.1em;
+}
+
+@media(max-width: 700px){
+  .home .banner{
+    background-position-x: -100px;
+    background-position-y: -120px;
+  }
+  .home-banner-btn{
+    margin-top: 10px;
+    max-width: 290px;
+  }
+  .banner .caption h1{
+    padding: 0 40px;
+}
+}
```

Comment:
This requires adding HTML code to the content area of Wordpress, and therefore breaks the clean separation rule of content and layout. You need to abstract the intent of the changes and add them to the theme itself as features, rather than having custom CSS for specific content and putting corresponding HTML in the content. I know it is easier to do it this way, but over time it will become unmaintainable.

Reply:
I remember briefly discussing this process with you a while back, but I might need a refresher. Maybe we could set up a meeting so I can have some support my first time adding a theme option?